indexing-engine/changelog.mdx

```diff
@@ -12,6 +12,17 @@ The dates on this page represent when features were first released for the cloud
 version of pganalyze. For pganalyze Enterprise Server releases, we indicate on each
 [release page](/docs/enterprise/releases) which release of the Indexing Engine is included.
 
+## 2022-08-09
```

Comment:
Should we back date this to the change where it was made? (or at least to the time of the last newsletter on August 3, where we also announced this?)